node/node.go

```diff
@@ -231,7 +230,7 @@ func MakeFull(log logging.Logger, rootDir string, cfg config.Local, phonebookAdd
 		Local:          node.config,
 		Network:        gossip.WrapNetwork(node.net, log),
 		Ledger:         agreementLedger,
-		BlockFactory:   blockFactory,
+		BlockFactory:   blockFactory{TransactionPool: node.transactionPool},
```

Comment:
If you're doing this, why don't you eliminate the blockFactoryImpl completely and move it's functionality to the node ?

Reply:
Sounds good--removed.